src/switch_core_media.c

```diff
@@ -5125,6 +5177,11 @@ static switch_status_t check_ice(switch_media_handle_t *smh, switch_media_type_t
 								interval = 5000;
 							}
 							switch_rtp_set_rtcp_passthru_timeout(engine->rtp_session, interval);
+						} else {
+							if (switch_media_handle_test_media_flag(smh, SCMF_RTCP_AUDIO_PASSTHRU_TIMEOUT_MSEC)) {
```

Comment:
I don't get this statement here, above if condition check for channel variable and rtcp_audio_passthru_timeout_msec media params. If this is both empty then val will also be empty here.

Reply:
It's direct merge from Damir's PR. Removing this would leave us with no use of `SCMF_RTCP_AUDIO_PASSTHRU_TIMEOUT_MSEC`. Should we remove it then?

Reply:
Please check with @damirn whats the purpose of this statement